packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -338,7 +338,7 @@ export default class ConfigParser {
                         path.dirname(this.#configFilePath)
                     )
                 )
-            } else if (globMatchedFiles.length) {
+            } else if (this.cliSpecArgContainsPathstoSpecs(cliArgFileList) && globMatchedFiles.length) {
```

Comment:
No need to put this into a separated function.
```suggestion
            } else if (cliArgFileList[0].includes(path.sep) && globMatchedFiles.length) {
```

Reply:
Also can you elaborate why this is needed? Maybe add a comment?

Reply:
The codebase is new to me. And I explored numerous options in search of a robust solution. It is plausible that, with the recent modifications, the necessity of this condition may have diminished. I'll double check this again. However, its presence definitely does not break anything))